### ISSUE TITLE
“classes” item properties not working

### ISSUE THREAD

**Issue by edsel77** [richardtallent/vue-simple-calendar#163]
I want to implement dynamic classes for each event items. Ex. background-color, etc. But the classes property is not working.

Reference Inquiry: https://stackoverflow.com/questions/64618986/vue-simple-calendar-classes-item-properties-not-working

**Comment by richardtallent** [comment]
This appears to be a bug in the documentation. The `classes` property is actually expected to be an *array of strings*, not a single string. *E.g.:

```
classes: ['bg-red']
```

Please try that and confirm if it fixes your issue.

BTW, version 4.1.0 is several versions behind, there have been a number of bug fixes and enhancements. I recommend upgrading if possible.

**Comment by edsel77** [comment]
setting string classes in an array does not work on version 4.4.0
upgrading to version 5.5.0, no events are showing in the calendar
may I know if the version 5.0.0 is ok use?

**Comment by richardtallent** [comment]
Did you follow the changelog instructions for upgrading to 5.0? There are breaking changes.

If you're still having issues after upgrading to v5.0, please include a URL where I can see the issue, using CodePlex or a similar service.

**Comment by richardtallent** [comment]
I've tested 5.0 to ensure that `classes` works for items. And once I dug in a little further, I found that it **does** actually work with both arrays of strings and just strings.

The `App.vue` file here now shows an example of using this property -- it randomly assigns a `classes` value of `custom-date-class-red` for some of the random events it creates, and has CSS to use that class to override the default Google Calendar theme color.
